### PR TITLE
Click accept on popup which shows when opening first channel

### PR DESCRIPTION
**Pull request by karoletrych**
Click accept on popup which shows when opening first channel.

### CODE REVIEW

File: download.py [hunk 1]
```diff
@@ -177,6 +177,12 @@ def check_gecko_driver():
                     print('[%d/%d] Channel %s needs VPN to be grabbed, skipping' % (item_n + 1, len(page_links), channel_list[item_n]), file=sys.stderr)
                     break
 
+                # close popup if it shows up
+                try:
+                    popupAcceptButton = driver.find_element_by_css_selector(POPUP_ACCEPT_CSS_SELECTOR).click()
```
Comment (interlark) [inline download.py:182]: Why you keep this variable?

File: download.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 from os import path
 
 IFRAME_CSS_SELECTOR = '.iframe-container>iframe'
-
+POPUP_ACCEPT_CSS_SELECTOR = '#qc-cmp2-ui > div.qc-cmp2-footer.qc-cmp2-footer-overlay.qc-cmp2-footer-scrolled > div > button.sc-ifAKCX.ljEJIv'
```
Comment (interlark) [inline download.py:23]: This selector is based on auto generated classname of styled component, so it's pretty unstable way of detecting "AGREE" button.
You better try something like this:
```python
driver.find_element_by_xpath("//button[contains(text(),'AGREE')]").click()
```

**Review by interlark — COMMENTED** [review]
Hmm, looks like in some countries, like UK, there is a blocking "privacy" modal dialog.
Please rewrite closing the dialog with **XPath method** I suggested in the review.

File: download.py (hunk 1 above)
Comment (karoletrych) [reply download.py]: Yup, not needed. Removed it.

File: download.py (hunk 2 above)
Comment (karoletrych) [reply download.py]: Changed it to xpath.

**Pull request merged**